### PR TITLE
modified output to display only lego colors (#2)

### PR DESCRIPTION
**Pull request by alexdeloy**
I added the original lego colors to the output.
Still needs a switch/parameter to use only lego colors.

### CODE REVIEW

File: legofy/__init__.py [hunk 1]
```diff
@@ -64,6 +129,7 @@ def makeLegoImage(baseImage, brick, width=30, height=30):
     for x in range(baseWidth):
         for y in range(baseHeight):
             bp = basePoa[x, y]
+            bp = getNearestColor(bp)
```
Comment (nabijaczleweli) [inline legofy/__init__.py]: You're assigning to the same verbibol twice, is that intended?

Comment (alexdeloy) [inline legofy/__init__.py]: my bad, should be one assignment of course

**Comment by nabijaczleweli** [comment]
As you can clearly see, you don't import `math`

**Comment by JuanPotato** [comment]
Why I haven't merged this: I want to find a way to use dithering instead of just trying to manually do it.

**Comment by ewjoachim** [comment]
Same comment as [on the other PR](https://github.com/JuanPotato/Legofy/pull/64#issuecomment-155430964)

**Comment by JuanPotato** [comment]
better merge has been merged

**Pull request closed without merging**